client/ayon_core/modules/deadline/plugins/publish/collect_default_deadline_server.py

```diff
@@ -33,15 +32,17 @@ def process(self, context):
         deadline_settings = context.data["project_settings"]["deadline"]
         deadline_server_name = deadline_settings["deadline_server"]
 
-        deadline_webservice = None
+        dl_ws_item = None
```

Comment:
Can we improve this variable name or description. Now it's an "item" but I have no idea what it is. In settings it's called `deadline_settings["deadline_urls"]` and in the collector and deadline module it's name "deadline_server_info".

So it's deadline url, server info or an item. An item in that list is the least descriptive.

This would be slightly better: `server_info: Dict[str, Any] = deadline_module.servers_info.get(server_name)`